src/components/qTip/theme.css

```
@@ -23,11 +23,12 @@
   border-right: 0;
   border-top-left-radius: 4px;
   border-bottom-left-radius: 4px;
+  transform: translateX(100%) translateX(calc(-1 * var(--tip-width) + 1px)); /* split into 2 separete translate, otherwise IE can't handle it */
   transition: all var(--animation-duration) var(--animation-curve-default);
   pointer-events: all;
 
-  .is-closed & {
-    transform: translateX(100%) translateX(calc(-1 * var(--tip-width) + 1px)); /* split into 2 separete translate, otherwise IE can't handle it */
+  .is-active & {
+    transform: translateX(0);
```

But then ofc you also need to change this
```suggestion
    transform: translate3d(0, 0, 0);
```

---

see comment above